user.c

```diff
@@ -0,0 +1,228 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+typedef struct
+{
+    int userID;
+    char userName[50];
+    int userAge;
+} Person;
+
+int checkUserID(int id)
+{
+    int result = 0; // Default result: User ID not found

```

Comment:
There is no need for these comments in the program.
Do not add comment until there is any complex logic with is difficult to express via code.